### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by Garfield96**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    schedule:
+      interval: "monthly"
+  - package-ecosystem: "docker"
+    directory: "/nginx/"
+    schedule:
+      interval: "monthly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (alexanderstephan) [inline .github/dependabot.yml:21]: Maybe we could terminate this file with a new line for better compatibility.

**Review by alexanderstephan — COMMENTED** [review]
Looks good to me, thanks! :+1: Just one minor comment from my side. 
I will let @joschahenningsen have the final word here of course.

File: .github/dependabot.yml (hunk 1 above)
Comment (Garfield96) [reply .github/dependabot.yml]: Fixed.

**Review by joschahenningsen — APPROVED** [review]
Looks good, thanks :beers:

**Pull request merged**